packages/plugin-e2e/src/fixtures/commands/createAppConfigPage.ts

```diff
@@ -0,0 +1,18 @@
+import { TestFixture } from '@playwright/test';
+import { PluginFixture, PluginOptions } from '../../api';
+import { PluginConfigPageArgs } from '../../types';
+import { PlaywrightCombinedArgs } from '../types';
+import { AppConfigPage } from '../../models/pages/AppConfigPage';
+
+type CreateAPPConfigPageFixture = TestFixture<
+  (args: PluginConfigPageArgs) => Promise<AppConfigPage>,
+  PluginFixture & PluginOptions & PlaywrightCombinedArgs
+>;
+
+export const createAppConfigPage: CreateAPPConfigPageFixture = async (ctx, use) => {
+  await use(async (args) => {
+    const page = new AppConfigPage(ctx, args);
+    await page.goto();
+    return page;
+  });
+};
```

Comment:
Can we find another way of passing/reading the plugin id without require the plugin developer to pass it in the test?

We could have read if from plugin.json or add it to the configuration. But how do we manage nested plugins in that case?

Do we have a use case where you would like to test the config page for another plugin than the one you are currently developing? Probably not?